Slide for Reddit/AlbumViewController.swift

```diff
@@ -22,14 +22,14 @@ class AlbumViewController: SwipeDownModalVC, UIPageViewControllerDataSource, UIP
     var bottomScroll = UIScrollView()
     var failureCallback: ((_ url: URL) -> Void)?
     var albumHash: String = ""
-    var galleryItems: List<String> = List<String>()
+    var galleryItems: [String] = [String]()
```

Comment:
I think you can just write it as this:
```suggestion
    var galleryItems: [String] = []
```